src/RevivalPMMP/PEXProtection/Main.php

```diff
@@ -16,179 +16,206 @@
+     * @param array $args
+     * @return bool
+     */
+    public function onCommand(CommandSender $p, Command $cmd, string $label, array $args) : bool {
+        if($cmd->getName() === "pexprotection") {
+            if($p->hasPermission("pexprotection.command") && $p instanceof Player) {
+                if(isset($args[1])) {
+
+                    switch($args[0]) {
+                        case "add":
+                        case "set":
+                        case "create":
+                        case "make":
+                            if(!$this->isCenterBlock($args[1])) {
+                                if(isset($args[2])) {
+                                    $p->sendMessage(TF::AQUA . "Tap a block to add a protection center block with the name " . $args[1] . "!");
+                                    $this->tapping = array();
+                                    $this->tapping[$p->getName()] = $p->getName();
+                                    $this->level = $p->getLevel()->getName();
+                                    $this->blockName = $args[1];
+                                    $this->radius = $args[2];
+                                    if(isset($args[3])) {
+                                        $this->allMobs = (strcmp(strtolower($args[3]), "true") == 0) ? true : false;
+                                    } else {
+                                        if ($this->getConfig()->exists("All-Mobs")) {
+                                            $this->allMobs = $this->getConfig()->get("All-Mobs");
+                                        } else {
+                                            $this->allMobs = true;
+                                        }
+                                    }
+                                } else {
+                                    $p->sendMessage(TF::RED .  "You have to define a radius for the protection center block!");
+                                }
+                            } else {
+                                $p->sendMessage(TF::RED . "A protection center block with that name already exists");
+                            }
+                            return true;
+                        
+	                    case "disableworld":
+	                    case "world":
+	                    	if(isset($args[1])) {
+			                    $worlds = $this->getConfig()->get("Disabled-Worlds");
+			                    $worlds[] = $args[1];
+			                    $this->getConfig()->set("Disabled-Worlds", $worlds);
+			                    $this->saveDefaultConfig();
+                                $p->sendMessage(TF::GREEN . "Successfully added a world to disable monster spawning.");
+		                    }
+		                    return true;
+	                    
+                        case "delete":
+                        case "del":
+                        case "remove":
+                        case "rem":
+                        case "clear":
```

Comment:
Why do these sub commands now require the user to be a player?

Reply:
Lack of foresight on my part. Added to the update list.